docs/Quickstart.md

```diff
@@ -0,0 +1,227 @@
+
+    <details>
+    <summary>More information</summary>
+
+    1. You will see structure of current folder like below:
+
+        ```text
+            │  .env
+            │  .gitignore
+            │  deployment.debug.template.json
+            │  deployment.template.json
+            │
+            ├─.vscode
+            │      launch.json
+            │
+            └─modules
+                └─filtermodule
+                    │  .gitignore
+                    │  Dockerfile.amd64
+                    │  Dockerfile.amd64.debug
+                    │  Dockerfile.arm32v7
+                    │  Dockerfile.windows-amd64
+                    │  filtermodule.csproj
+                    │  module.json
+                    │  Program.cs
+        ```
+
+    2. Open `.env` file, you will see the `IOTHUB_CONNECTION_STRING` and `DEVICE_CONNECTION_STRING` environment variables filled correctly.
+    3. Open `deployment.template.json` file
+        1. You will see below section in the modules section:
+
+            ```json
+            "filtermodule": {
+                "version": "1.0",
+                "type": "docker",
+                "status": "running",
+                "restartPolicy": "always",
+                "settings": {
+                    "image": "${MODULES.filtermodule}",
+                    "createOptions": {}
+                }
+            }
+            ```
+
+        2. Two default routes are added:
+
+            ```json
+            "routes": {
+                "sensorTofiltermodule": "FROM /messages/modules/tempSensor/outputs/temperatureOutput INTO BrokeredEndpoint(\"/modules/filtermodule/inputs/input1\")",
+                "filtermoduleToIoTHub": "FROM /messages/modules/filtermodule/outputs/* INTO $upstream"
+            }
+            ```
+
+    4. You will see privacy statement like below:
+
+        ```text
+        Welcome to iotedgedev!
+        -------------------------
+        Telemetry
+        ---------
+        The iotedgedev collects usage data in order to improve your experience.
+        The data is anonymous and does not include commandline argument values.
+        The data is collected by Microsoft.
+        
+        You can change your telemetry settings by updating 'collect_telemetry' to 'no' in ~/.iotedgedev/setting.ini
+        ```
+
+    </details>
+
+1. **Build IoT Edge module images**
+
+    `sudo iotedgedev build`
+
+    > This step will build user modules in deployment.template.json targeting amd64 platform.
+
+    <details>
+    <summary>More information</summary>
+
+    1. You will see a "BUILD COMPLETE" for each module and no error messages in the terminal output.
+    1. Open `config/deployment.amd64.json` file, you will see the module image placeholders expanded correctly.
+    1. Run `sudo docker image ls`, you will see the module images you just built.
+
+    </details>
+
+### Set up and start modules on a virtual machine
```

Comment:
>### [](http://example.com/codeflow?start=0&length=3)

I think it would add clarity to have a section 3. Run Edge modules. We should clearly state that there are multiple ways to do so - VM/physical device is recommended, simulator is also available. Some folks are new tot he simulator and may think that they need to setup the VM AND the simulator.